ddl/src/main/java/com/adaptivescale/rosetta/ddl/targets/kinetica/KineticaDDLGenerator.java

```diff
@@ -67,6 +67,9 @@ public String createTable(Table table, boolean dropTableIfExists) {
             stringBuilder.append(dropTable(table));
         }
 
+        String tableType = extractTableType(table);
+
+        createParams.put("tableType", tableType);
```

Comment:
Do we know what happens for other DB Types? This attribute is not displayed at all right?